cli/internal/context/context.go

```diff
@@ -305,21 +304,25 @@ func (c *Context) populateTopologicGraphForPackageJSON(pkg *fs.PackageJSON, root
 	return nil
 }
 
-func (c *Context) parsePackageJSON(buildFilePath string) error {
+func (c *Context) parsePackageJSON(repoRoot fs.AbsolutePath, pkgJSONPath fs.AbsolutePath) error {
 	c.mutex.Lock()
 	defer c.mutex.Unlock()
 
 	// log.Printf("[TRACE] reading package.json : %+v", buildFilePath)
```

Comment:
(request for change)

Let's drop this log which now logs something that doesn't exist.